spammer/src/spammer_profile.toml

```diff
@@ -0,0 +1,10 @@
+#[generation]
+# Generation weights in terms of: BaseBasicTransactions, BurstyBasicTransactions, Vesting Contracts
+weights = [ 7, 2, 1]
+
+# Probability of sending from existing accounts to other accounts
+many_to_many = 0.5
+
+# The Base of BasicTransactions that are generated per block. 
+# Bursty transactions are generated in the range [tpb * 10, tpb * 20]
+tpb = 50
```

Comment:
Looks like this could go to our config file without needing to create another configuration source

Reply:
Since the spammer is some testing-only tool, I decided to not mix our config file with spammer stuff..